src/components/nve-label/nve-label.demo.ts

```diff
@@ -74,18 +74,40 @@ const table = html`
         </td>
       </tr>
       <tr>
-        <td>Innhold i slot</td>
+        <td>Med info-ikon på venstre siden</td>
+        <td>
+          <nve-label value="Svev over ikonet" iconLeft>
+            <div slot="tooltip">Hjelpetekst i <strong>HTML</strong></div>
+          </nve-label>
+        </td>
+        <td>
+          <nve-label light value="Svev over ikonet" iconLeft>
+            <div slot="tooltip">Hjelpetekst i <strong>HTML</strong></div>
+          </nve-label>
+        </td>
+      </tr>
+      <tr>
+        <td>Med svart info-ikon</td>
         <td>
-          <nve-label>
-            Ledetekst i <i>HTML</i>
+          <nve-label value="Svev over ikonet" iconColor="black">
```

Comment:
Du trenger ikke ha tooltip i slot for å demonstrere farge på ikonet?
Eksemplet blir enklere om du heller bruker tooltip-property

Reply:
Kan vi vise en annen type ikone da? Jeg trodde tooltip info ikon var eneste som vises i nve-label